README.md

```diff
@@ -21,7 +21,17 @@ To start development of a new connector:
 
 1. Copy this repository code to your repository
 2. Replace all entries **template** with **yourConnectorName** and **Template** with **YourConnectorName**
-3. Start implementing your connector specific logic replacing TODOs
+3. Start implementing your connector specific logic replacing TODOs. Note that field values in cacheItemStructure of [envResponseBody](src\routes\env\schema.ts) should be in the following format:
+
+   - First letter of the first name word should be capitalized.
```

Comment:
Let's follow the way eslint rules explain themselves:
- State the rule
- Show a bad example
- Show a good example

e.g.:

```md
- The first letter should be capitalized
  - Wrong: "author"
  - Correct: "Author"
- Words after the first word should not be capitalized
  - Wrong: "Group Name"
  - Correct: "Group name"
```

Reply:
done